README.md

```diff
@@ -34,22 +41,46 @@ environments. If you have conda installed locally, you can run the following to
 create and activate a new environment.
 
 ```
->> conda create -n <env_name> python=3.10
->> conda activate <env_name>
+conda create -n <env_name>
+conda activate <env_name>
 ```
 
 Once you have created a new environment, you can install this project for local
 development using the following commands:
 
 ```
->> git submodule update --init --recursive
->> conda install -c conda-forge cxx-compiler
->> pip install -e .'[dev]'
->> pre-commit install
->> conda install pandoc
+git clone git@github.com:lephare-photoz/lephare.git
+git submodule update --init --recursive
+conda install -c conda-forge cxx-compiler #needed for MACOSX mostly
+pip install -e .'[dev]'
+pre-commit install
+conda install pandoc
+```
+The installation can then be tested by running `python -m pytest tests` from the root directory of the repository.
+
+In order to obtain the same directory hierarchy as the legacy fortran or c++ version, with the data files inside the
+repository directory structure, the following can be executed:
+
+```
+from lephare import data_retrieval as dr
+dr.get_auxiliary_data(lephare_dir=os.environ["LEPHAREDIR"], clone=False)
```

Comment:
LEPHAREDIR would have to be set to the code repo prior to the import. e.g.

import os
oe.environ["LEPHAREDIR"]="/path/to/lephare/" # Can we get this automatically and check that it is not pip installed?
from lephare import data_retrieval as dr
dr.get_auxiliary_data(lephare_dir=os.environ["LEPHAREDIR"], clone=False)

Reply:
You can get it with inspect but it is a bit drawn out:

```
import inspect
import lephare
import importlib
lephare_dir = inspect.getfile(lephare).replace('src/lephare/__init__.py','')
os.LEPHAREDIR=lephare_dir
importlib.reload(lephare)
from lephare import data_retrieval as dr
dr.get_auxiliary_data(lephare_dir=lephare_dir, clone=False)
```

And I guess if it was pip installed it would send all the files to the conda folder.

Reply:
yes I am not sure that we need to automatize everything. Here we are providing information for developers. Setting LEPHAREDIR to the top dir is what has been done systematically in the past.